src/docfx/build/legacy/files/LegacyMarkdown.cs

```diff
@@ -64,11 +77,17 @@ internal static class LegacyMarkdown
 
         private static JObject GenerateLegacyRawMetadata(PageModel pageModel, Docset docset, Document file, GitRepoInfoProvider repo)
         {
+            var depotName = docset.Config.Name;
+            var items = depotName.Split('.');
```

Comment:
use `string.IndexOf`, as docsetName may contain `.`

Reply:
this can be merged with @OsmondJiang 's PR